IncludeToolbox/IncludeFormatter/IncludeLineInfo.cs

```diff
@@ -92,6 +92,8 @@ public static IncludeLineInfo[] ParseIncludes(string text, bool removeEmptyLines
                 if (multiLineCommentStart > -1 && multiLineCommentStart < includeOccurence) // Multi line comment started in same line, but before #include.
                     continue;
 
+                if (lines[line].Contains(".inl>") || lines[line].Contains(".inl\"")) // Don't remove inline files.
```

Comment:
Hmmm yeah I can see why you're doing this - linker errors between the try & error removal was too aggressive. But this also changes the way sorting is done, so I'd rather make this optional via a flag that is only set for the try&error removal if you don't mind.